webcola/webcola.d.ts

```diff
@@ -1,71 +1,1097 @@
-// Type definitions for webcola 3.0.1
+// Type definitions for webcola 3.1.1
 // Project: https://github.com/tgdwyer/WebCola
-// Definitions by: Qinfeng Chen <https://github.com/qinfchen>
```

Comment:
please keep original author. add your name to end of line.
like this https://github.com/borisyankov/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3